flink-state-backends/flink-statebackend-changelog/src/main/java/org/apache/flink/state/changelog/ChangelogStateBackend.java

```diff
@@ -222,8 +223,11 @@ public StateBackend configure(ReadableConfig config, ClassLoader classLoader)
                                 ttlTimeProvider,
                                 changelogStorage.createWriter(operatorIdentifier, keyGroupRange),
                                 baseState,
+                                new AsyncExceptionHandler(env),
                                 env.getMainMailboxExecutor(),
-                                env.getAsyncOperationsThreadPool()));
+                                env.getAsyncOperationsThreadPool(),
+                                env.getCheckpointStorage()
+                                        .createCheckpointStorage(env.getJobID())));
```

Comment:
I think it's better to pass `CheckpointStorageAccess` created in `StreamTask` as an argument, because:
- The returned object is supposed to be used by multiple backends so it doesn't seem like a backend responsibility to create it
- the call can also potentially create a new storage. 
- if it has to be closed then it will be unclear who is doing it (existing backends using it are deprecated) 

(at least add a todo)

Reply:
After reading the code, I agree that we should follow the current contract that sharing the same `CheckpointStorageAccess`, so that the ownership still kept in `StreamTask`